.github/workflows/deploy.yml

```diff
@@ -36,11 +36,22 @@ jobs:
       Version: latest
     secrets: inherit
 
+  approval-step:
+    # This step is here just to get approval, without it GHA will proceed onto
+    # the next steps, check the concurrency rules and cancel them in certain
+    # cases. The resolution order appears to be:
+    # 1. Move to next job
+    # 2. Check concurrency (cancelling here is not what we want)
+    # 3. Ask for approval
+    needs: qa-us-west-1
+    with:
+      Environment: prod
```

Comment:
I think we need to test this to find out for sure, but I have a feeling we will hit 2 approval stages.

- One for the new approval-step job being added
- One for the prod deploys

However, I could be wrong.

Reply:
I suspect you are right, but should we try this first as it involves less shenanigans? And then try the "approval" environment as a fallback?

Reply:
If we test this out at least we'll verify our understanding of how this GHA behavior works.

Reply:
I think the `approval` environment will work around the problem and means we don't have to modify the function being called. We can just remove the protection rules.

I save give the `approval` route a test.